yarn.lock

```diff
@@ -9634,13 +9639,12 @@ postcss-value-parser@^4.1.0, postcss-value-parser@^4.2.0:
   integrity sha512-1NNCs6uurfkVbeXG4S8JFT9t19m45ICnif8zWLd5oPSZ50QnwMfK+H3jv408d4jw/7Bttv5axS5IiHoLaVNHeQ==
 
 postcss@^7.0.17:
```

Comment:
The v7 version comes from `stylelint-config-idiomatic-order`. Maybe we should import this package to our tree directly and depend in `stylelint-order` instead, so that we can upgrade more easily that one?

Reply:
Yeah, it should be possible to do so.